superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -212,6 +213,7 @@ class SqlEditor extends React.PureComponent {
     this.setState({ height: this.getSqlEditorHeight() });
 
     window.addEventListener('resize', this.handleWindowResize);
+    window.addEventListener('beforeunload', this.onBeforeUnload.bind(this));
```

Comment:
same thing.. don't think you need the bind.

Reply:
thanks fixed by efb86e51af1959de182faf486a6877164ebd933f

can you please approve the CI workflow so github actions will run again? :bowing_man: